lib/Elastica/Client.php

```diff
@@ -38,6 +38,8 @@ class Client
         'log' => false,
         'retryOnConflict' => 0,
         'bigintConversion' => false,
+        'username' => null,
```

Comment:
I was first thinking if we should better add this inside the connections. But that would mean setting it up for each connection. I think it can be expected that all elasticsearch nodes have the same users so I'm happy to have it hear.